charts/psmdb-operator/values.yaml

```diff
@@ -9,6 +9,13 @@ image:
   tag: 1.14.0
   pullPolicy: IfNotPresent
 
+# disableTelemetry: according to
+# https://docs.percona.com/percona-operator-for-mongodb/telemetry.html
+# this is how you can disable telemetry collection
+# default is false which means telemetry will be collected
+# NOTE: "true" or "false" must be a string
```

Comment:
Please just remove this line since it is not needed any more (the note about the string).

Reply:
done